### PR TITLE
operator registry-operator (0.3.0)

### PR DESCRIPTION
**Pull request by thepetk**
Thanks for submitting your Operator. Please check the below list before you create your Pull Request.

### New Submissions

* [ ] Are you familiar with our [contribution guidelines](https://github.com/redhat-openshift-ecosystem/community-operators-prod/blob/main/docs/contributing-via-pr.md)?
* [ ] Are you familiar with our [operator pipeline](https://github.com/redhat-openshift-ecosystem/community-operators-prod/blob/main/docs/community-operator-pipeline.md)?
* [ ] Have you tested your Operator with all Custom Resource Definitions [packaging](https://github.com/redhat-openshift-ecosystem/community-operators-prod/blob/main/docs/packaging-operator.md)?
* [ ] Have you tested your Operator in all supported [installation modes](https://github.com/redhat-openshift-ecosystem/community-operators-prod/blob/main/docs/operator-ci-yaml.md##Operator-versioning)?
* [ ] Have you considered whether you want to use [semantic versioning order](https://github.com/operator-framework/community-operators/blob/master/docs/operator-ci-yaml.md#semver-mode)?
* [ ] Is your submission [signed](https://github.com/redhat-openshift-ecosystem/community-operators-prod/blob/main/docs/contributing-prerequisites.md#sign-your-work)?
* [ ] Is operator [icon](https://github.com/redhat-openshift-ecosystem/community-operators-prod/blob/main/docs/packaging-operator.md#operator-icon) set?

### Your submission should not

* [ ] Add more than one operator bundle per PR
* [ ] Modify any operator
* [ ] Rename an operator
* [ ] Modify any files outside the above mentioned folders
* [ ] Contain more than one commit. **Please squash your commits.**

### Operator Description must contain (in order)

1. [ ] Description of the managed Application and where to find more information
2. [ ] Features and capabilities of your Operator and how to use it
3. [ ] Any manual steps about potential pre-requisites for using your Operator

### Operator Metadata should contain

* [ ] Human readable name and 1-liner description about your Operator
* [ ] Valid [category name](https://github.com/redhat-openshift-ecosystem/community-operators-prod/blob/main/docs/packaging-operator.md#categories)<sup>1</sup>
* [ ] One of the pre-defined [capability levels](https://github.com/operator-framework/operator-courier/blob/4d1a25d2c8d52f7de6297ec18d8afd6521236aa2/operatorcourier/validate.py#L556)<sup>2</sup>
* [ ] Links to the maintainer, source code and documentation
* [ ] Example templates for all Custom Resource Definitions intended to be used
* [ ] A quadratic logo

Remember that you can preview your CSV [here](https://operatorhub.io/preview).

--

<sup>1</sup> If you feel your Operator does not fit any of the pre-defined categories, file an issue against this repo and explain your need

<sup>2</sup> For more information see [here](https://sdk.operatorframework.io/docs/overview/#operator-capability-level)

### CODE REVIEW

**Comment by michael-valdron** [comment]
https://github.com/redhat-openshift-ecosystem/community-operators-prod/pull/5056 unblocks @thepetk as an authorized contributor to the devfile registry operator.

**Review by michael-valdron — COMMENTED** [review]
> #5056 unblocks @thepetk as an authorized contributor to the devfile registry operator.

@thepetk Needs rebase to the change I did to include your user under `ci.yaml`.

File: operators/registry-operator/0.3.0/metadata/annotations.yaml
Comment (michael-valdron) [inline operators/registry-operator/0.3.0/metadata/annotations.yaml:1]: For the OpenShift OperatorHub we'll need to include an additional annotation to specify OpenShift version support, for example: https://github.com/redhat-openshift-ecosystem/community-operators-prod/blob/13f437d15286acf06ce48254293fef77c5ffcdab/operators/registry-operator/0.2.0/metadata/annotations.yaml#L12-L13

Comment (thepetk) [reply operators/registry-operator/0.3.0/metadata/annotations.yaml:1]: @michael-valdron I've added support for `v4.12-v4.15` based on what we have https://github.com/devfile/registry-operator/blob/main/README.md and the fact that we didn't include (IIRC) any breaking changes from 0.2.0 to 0.3.0.

**Pull request merged**